inductiva/fluids/scenarios/coastal_area/coastal_area.py

```diff
@@ -19,22 +21,103 @@
 SWASH_TEMPLATE_SUBDIR = "swash"
 SWASH_CONFIG_TEMPLATE_FILENAME = "input.sws.jinja"
 SWASH_CONFIG_FILENAME = "input.sws"
+SWASH_BATHYMETRY_FILENAME = "bathymetry.bot"
+
+
+class Bathymetry:
```

Comment:
I love this class. 
A few of comments:

1)  it's not clear if the need for the x_range and y_range is just because we are using txt files. Is there another format where the range info is provided so that we don't need to provide it in the constructor?

2) I would add a clip() method, as well as potentially other utility methods. This could be done later.

3) I would consider adding a method (either in this class or as an util method) to create random bathymetries. I think we need to really use different bathymetries to test this class.

Reply:
> 1. it's not clear if the need for the x_range and y_range is just because we are using txt files. Is there another format where the range info is provided so that we don't need to provide it in the constructor?

Yes, there are other formats where the range info is provided. We can add support for those in the future. For now the goal was to read the bathymetry from a text file (the ones SWASH users load bathymetries from, in any case).

> 2. I would add a clip() method, as well as potentially other utility methods. This could be done later.

Agreed, to be done later.

> 3. I would consider adding a method (either in this class or as an util method) to create random bathymetries. I think we need to really use different bathymetries to test this class.

That's in our (short-term) plans too.